shardingsphere-features/shardingsphere-sharding/shardingsphere-sharding-distsql/shardingsphere-sharding-distsql-handler/src/main/resources/META-INF/services/org.apache.shardingsphere.infra.distsql.query.DistSQLResultSet

```diff
@@ -29,3 +29,4 @@ org.apache.shardingsphere.scaling.distsql.handler.ShowScalingJobStatusQueryResul
 org.apache.shardingsphere.scaling.distsql.handler.ShowScalingCheckAlgorithmsQueryResultSet
 org.apache.shardingsphere.sharding.distsql.handler.query.UnusedShardingAlgorithmsQueryResultSet
 org.apache.shardingsphere.sharding.distsql.handler.query.UnusedShardingKeyGeneratorsQueryResultSet
+org.apache.shardingsphere.sharding.distsql.handler.query.ShardingTableRulesUsedKeyGeneratorQueryResultSet
```

Comment:
Blank line is required at the end of file.

Reply:
@yx9o thank you for your suggestion. learn lots from you

Reply:
@LeeGuoPing Learn from each other and work together.